src/Infrastructure/Persistence/Migrations/ApplicationDbContextModelSnapshot.cs

```diff
@@ -137,7 +137,7 @@ protected override void BuildModel(ModelBuilder modelBuilder)
                             b1.Property<Guid>("HeroId")
                                 .HasColumnType("uniqueidentifier");
 
-                            b1.Property<int>("Id")
+                            b1.Property<int>("__synthesizedOrdinal")
```

Comment:
I haven't seen this before.  Is this the name of the column in the DB?

Reply:
As per our conversation, I suspect this is due to the change in EF core tools version. As discussed I removed the migration and ran it again using 9 rc and confirmed there is no difference with this version.

According to the docs, breaking changes were noted in EF Core 8 that may be responsible for this, but it looks like it's related to strings, not ints, or to CosmosDB, so I'm a little confused.

https://learn.microsoft.com/en-us/ef/core/what-is-new/ef-core-8.0/breaking-changes

How do you want to proceed?

Reply:
I've had a look at the DB tables created from your branch and the column names look right.

<img width="304" alt="image" src="https://github.com/user-attachments/assets/e31a561e-23b9-452d-baf8-47f0cb6a2485">

Can we add some column length constraints into the EF Core config, so that we don't end up with `nvarchar(max)` columns?

I had a chat with Bryden about this some time ago, and we should endevour not to end up with max length columns.

Reply:
Donr